functest/get_balance_test.go

```diff
@@ -37,5 +38,5 @@ func TestGetBalance(t *testing.T) {
 func TestGetBalanceWrongRef(t *testing.T) {
 	_, err := getBalance(&root, testutils.RandomRef().String())
 	require.NotNil(t, err)
-	require.Contains(t, err.Error(), "index not found")
+	require.True(t, errors.IsNonRetryable(err), "index not found")
```

Comment:
The last param of `require.True` is the additional message to tests trace in case of fail. Why is it says `index not found`?

Reply:
We send API request here, so you don't get direct instance of `Retryable` error, you get some json with `error` field inside, and convert it to basic `error` interface for working with it like its object.

Reply:
reverted this part